openqaoa/problems/problem.py

```diff
@@ -14,35 +14,44 @@
 
 from abc import ABC, abstractmethod
 from collections import defaultdict, Counter
-from random import seed
-
 import networkx as nx
 import numpy as np
 import scipy
 import scipy.spatial
 import itertools
 
-from .helper_functions import convert2serialize, check_kwargs
+from .helper_functions import convert2serialize, convert_binary_to_ising
 from openqaoa.qaoa_parameters.operators import Hamiltonian
 
+SEED = 1234
+
 
 class Problem(ABC):
     @staticmethod
     @abstractmethod
-    def random_instance(**kwargs):
+    def random_instance(problem_size):
```

Comment:
We should add an explanation of this input parameter somewhere in the docstring.
Probably in every class since problem size in each problem class would/could mean different things.